docs/installation.md

```diff
@@ -31,7 +31,23 @@ git clone -b <latest-release-tag> https://github.com/cosmos/gaia
 cd gaia && make install
 ```
 
-> _NOTE_: If you have issues at this step, please check that you have the latest stable version of GO installed.
+If this command fails due to the following error message, you might have already set `LDFLAGS` prior to running this step.
+
+```
+# github.com/cosmos/gaia/cmd/gaiad
+flag provided but not defined: -L
+usage: link [options] main.o
+...
+make: *** [install] Error 2
+```
+
+Unset this environment variable and try again.
+
+```
+LDFLAGS="" make install
```

Comment:
There are some worthwhile flags we'd like to add, namely:

```
ldflags = -X github.com/cosmos/cosmos-sdk/version.Name=gaia \
		  -X github.com/cosmos/cosmos-sdk/version.ServerName=gaiad \
		  -X github.com/cosmos/cosmos-sdk/version.ClientName=gaiacli \
		  -X github.com/cosmos/cosmos-sdk/version.Version=$(VERSION) \
		  -X github.com/cosmos/cosmos-sdk/version.Commit=$(COMMIT) \
		  -X "github.com/cosmos/cosmos-sdk/version.BuildTags=$(build_tags_comma_sep)" \
			-X "github.com/cosmos/cosmos-sdk/client/rest.SwaggerTitle=$(SWAGGER_TITLE)" \
			-X github.com/cosmos/cosmos-sdk/client/rest.SwaggerHost=$(SWAGGER_HOST) \
			-X github.com/cosmos/cosmos-sdk/client/rest.SwaggerVersion=$(SWAGGER_VERSION)
```

Do we want to avoid these?

Reply:
I can quickly [add these to the makefile](https://github.com/cosmos/gaia/blob/5e817641bb5256b80dab6928f45a65ba6b907786/Makefile#L50-L55), if you want.

Reply:
bump @captain-kark

Reply:
I think [those values still get set](https://github.com/cosmos/gaia/blob/dffe3b2258edc4c0a0db92cc14517f95141eae99/Makefile#L60), the command that's been provided in the example is to remove any `LDFLAGS` that users [might have set in their environment that they didn't intend to use](https://github.com/captain-kark/dotfiles/blob/e81dfafb28faf7a9c0adfdfeabaae7a732ebbba7/.bash_profile#L18).

Reply:
You're correct. I tested and it works with `LDFLAGS="" make install`.